src/main/java/com/microsoft/jenkins/appservice/WebAppDeploymentRecorder.java

```diff
@@ -349,18 +350,18 @@ public FormValidation doVerifyConfiguration(@AncestorInPath final Item owner,
             }
         }
 
-        @JavaScriptMethod
-        public boolean isWebAppOnLinux(
-                final String azureCredentialsId,
-                final String resourceGroup,
-                final String appName) {
+        public String doIsWebAppOnLinux(@AncestorInPath Item owner,
```

Comment:
Will the renaming have any impact to our VSCode Jenkins plugin?

Reply:
I'm not sure. Is there any instructions how we can test against the VSCode plugin?

Reply:
Noted, let's follow up if there's any further issue